src/WatchModel.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+require_once 'src/Watch.php';
+
+class WatchModel
+{
+    public PDO $db;
+    
+    public function  __construct(PDO $db) 
+    {
+        $this->db = $db;
+    }
+
+
+    public function getAllWatches()
+    {
+        $query = $this->db->prepare('SELECT `watch`.`id`,`watch`.`brand`, `watch`.`model_name`, `watch`.`dial_colour`, `type`.`movement` FROM `watch` JOIN `type` ON `watch`.`watch_type` = `type`.`id`;');
```

Comment:
This line is getting quite long, you should break it down into multiple to make it easier to read